Directory.Build.targets

```diff
@@ -131,6 +131,17 @@
     <SuppressPackageTargetFrameworkCompatibility Include="$(UAPvNextTFM)" />
   </ItemGroup>
 
+  <!-- 
+    Import Test SDK props and targets.
+    This needs to happen here as the IsInvokableTestProject prop is required
+    which isn't available before Directory.Build.targets.
+    TODO: Fix VB test project reference to Test SDK.
+  -->
+  <Import Condition="'$(IncludeVSTestReferences)' == 'true' AND '$(Language)' != 'VB'" Project="$(PackagesDir)\microsoft.net.test.sdk\$(MicrosoftDotNetTestSdkVersion)\buildMultiTargeting\Microsoft.Net.Test.Sdk.props" />
+  <Import Condition="'$(IncludeVSTestReferences)' == 'true' AND ('$(BuildingNETCoreAppVertical)' == 'true' OR '$(BuildingNETFxVertical)' == 'true')" Project="$(PackagesDir)\microsoft.codecoverage\$(MicrosoftDotNetTestSdkVersion)\build\netstandard1.0\Microsoft.CodeCoverage.props" />
```

Comment:
Should we put this condition into a property to avoid duplication?

Reply:
Same as XUnitExtensions. Not going to do this now, let's consider it for a later clean up.